tools/fragpipe/fragpipe.xml

```diff
@@ -71,6 +71,9 @@
         <!-- Only generate a workflow, don't run FragPipe -->
         <param name="workflow_only" type="boolean" truevalue="yes" falsevalue="no" checked="false" label="Only generate the workflow output"/>
 
+        <!-- Galaxy admins may utilize this variable to for jobs requiring higher memory  -->
+        <param name="high_memory" type="boolean" checked="false" label="High memory job" help="This option is made available for Galaxy administrators to allocate more resources to FragPipe when selected. It may not be used by all systems - please contact your adminstrator to learn if your Galaxy instance does."/>
```

Comment:
Sorry I missed some discussion I guess. How should a user know that? What is the use-case here?
Is the TPV rule not working?

Reply:
The TPV rule checks for this parameter. So this parameter will not be used by all installations, which I noted to the user in the help section.

Reply:
I'm not sure there's an exact precedent for this. I see that hifiasm also passes the parameter it uses for TPV to the program's command-line args, whereas this parameter is only used by TPV. But I thought this was a straightforward way to allow a user to request more memory, if the request can be fulfilled.